exammode/migrations/0002_examsession_exam_module.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.3 on 2019-08-05 08:51
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course', '0044_auto_20190415_1807'),
+        ('exammode', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Looks like this migration could be combined into exammode/migrations/0001_initial.py. Is there any reason to separate them?

Reply:
Will be solved by generating migrations again before merge